### PR TITLE
Cleaner braille PR

### LINKED ISSUE #4: braille
Maybe this is getting too crazy, but I think we could make a more compact text-based layout with [braille characters](http://www.alanwood.net/unicode/braille_patterns.html).

I dunno if it's even useful, but I think it might be neat. 

Braille uses a pattern:

```
   ,___,
   |1 4|
   |2 5|
   |3 6|
   |7 8|
   '''''
```

or in binary-flags:

```
{0x01, 0x08},
{0x02, 0x10},
{0x04, 0x20},
{0x40, 0x80}
```

so in order to get walls, it's 2 chars per grid-square

N + S:

```
   ,___,   ,___,
   |1 4|   |1 4|
   |   |   |   |
   |   |   |   |
   |7 8|   |7 8|
   '''''   '''''
```
N + E:

```
   ,___,   ,___,
   |1 4|   |1 4|
   |   |   |  5|
   |   |   |  6|
   |   |   |  8|
   '''''   '''''
```

S + E:

```
   ,___,   ,___,
   |   |   |  4|
   |   |   |  5|
   |   |   |  6|
   |7 8|   |7 8|
   '''''   '''''
```

Out of interest, I think I will make a maze-to-braille function, and I can PR to add it, if it sounds good.

### PR DESCRIPTION
**Pull request by konsumer**
This is basically the same stuff in #5 (resolves #4)

I just fixed a typo in README from last commit, and also added notes about braille.

### CODE REVIEW

**Comment by konsumer** [comment]
I think I cleaned up all the typos, so this should be GTG. It's a totally separate script, so it's more of a demo for luamaze, but I could integrate it more, like I did with the love stuff, if you like. If I encapsulate the `utf8` stuff in the render function, it should be safe for all environments, like the way the love stuff is hiding in a function (so safe to import an in any version of modern lua, even without love.)

I'm hestitant to replace `to_string()` with it, because that's useful too, and if your terminal can't show braille (I think most can) it would not work. Also, it requires `utf8` module for luajit, which is easy enough to install, but maybe that should not be a default requirement.

File: source/braille.lua [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+   |7 8|   |7 8|
+   '''''   '''''
+add 0x2800 (braille base) like this:
+NE is
+0x2800 + 0x01 + 0x08
+0x2800 + 0x01 + 0x08 + 0x10 + 0x20 + 0x80
+⠉⢹
+]]
+
+function setbits(x, pa)
```
Comment (shironecko) [inline source/braille.lua:52]: I read up on bitwise operators in Lua and the situation seems... complicated. So I'm not against doing this fancy bit of math to emulate true bitwise op, but could you add a comment explaining the reasoning behind the inclusion of such a function as well as some implementation details?
I feel like people coming from languages that have proper support for bitwise operations might be confused by this.

File: source/braille.lua [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  return out
+end
+
+-- get CLI args
+local width = arg[1] or 10
+local height = arg[2] or 10
+local algo = arg[3] or "recursive_backtracker"
+local maze = Maze:new(width, height, true)
+
+-- maek sure random is more random
```
Comment (shironecko) [inline source/braille.lua]: minor: maek -> make

**Review by shironecko — CHANGES_REQUESTED** [review]

File: source/braille.lua (hunk 1 above)
Comment (konsumer) [reply source/braille.lua:52]: yeh, that was exactly why I did it. I was trying to combat not having a good cross-lua way to do bitwise. I will add a note.

**Comment by konsumer** [comment]
Ok, suggestions should be addressed. also added another note about love, as you don't need a separate `utf8` just need to `require` it.

**Review by shironecko — APPROVED** [review]

**Pull request merged**